tests/test_karma.py

```diff
@@ -40,26 +40,30 @@ def test_parse_karma_change(test_change, expected):
         ("EFG123", "ABC123", "CHANNEL42", -3),
     ],
 )
-def test_change_karma(mock_filled_db_session, test_changes, mock_slack_api_call):
-    session = db_session.create_session()
-    pre_change_karma = session.query(KarmaUser).get(test_changes[1]).karma_points
+@pytest.mark.usefixtures("conversations_info_fake_channel", "mock_filled_db_session")
+def test_change_karma(test_changes):
+    with database.session_manager() as session:
+        pre_change_karma = session.query(KarmaUser).get(test_changes[1]).karma_points
 
     karma = Karma(test_changes[0], test_changes[1], test_changes[2])
     karma.change_karma(test_changes[3])
 
-    post_change = session.query(KarmaUser).get(test_changes[1]).karma_points
+    with database.session_manager() as session:
```

Comment:
duplicated, could abstract out into a helper function

Reply:
does not work well with mocks

Reply:
ok